spec/rails_helper.rb

```diff
@@ -3,10 +3,12 @@
 require File.expand_path("../../config/environment", __FILE__)
 require "rspec/rails"
 require "capybara/rails"
+require 'helpers/helpers_spec.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.